src/zappy_server_src/core/entity/egg.c

```diff
@@ -18,6 +18,7 @@ egg_t *create_new_egg(const char *team_name)
     if (egg == NULL)
         return (NULL);
     egg->team_name = strdup(team_name);
+    egg->hatched = false;
```

Comment:
What's that ?

Reply:
State representation of the egg. Used by the GUI to display if the egg has hatched or not